py_gtfs_rt_ingestion/ingest.py

```diff
@@ -99,7 +100,15 @@ def ingest() -> None:
 
     total_filecount = 0
 
-    ingest_files(files)
+    # start rds writer process
+    metadata_queue = start_rds_writer_process()
+
+    ingest_files(files, metadata_queue)
+
+    # send shutdown signal to rds process and wait for finish
+    metadata_queue.put(None)
+    metadata_queue.close()
+    metadata_queue.join_thread()
```

Comment:
initial attempt at having a single process in `ingestion` to handle all rds operations. 

communication is handled through a `multiprocessing` `Queue` object. 

metadata information can just be sent on the Queue and the writer process will take care of the rest